docs/csaf_aggregator.md

```diff
@@ -134,12 +135,22 @@ insecure = true
   domain = "localhost"
 #  rate = 1.5
 #  insecure = true
+   category = "lister"
```

Comment:
changes here are in the wrong file, it gets inserted from docs/examples/aggregator.toml

Reply:
(Okay, the changes are also in  docs/examples/aggregator.toml, so it can stay in there, would just be unnecessary as it gets replaced.)